### PR TITLE
Add API to get the current graph version

### PR DESCRIPTION
**Pull request by roshandathathri**

### CODE REVIEW

**Review by tylershunt — APPROVED** [review]

**Comment by roshandathathri** [comment]
> Error: This repo is not allowlisted for Atlantis.

What is this!!?

**Comment by tylershunt** [comment]
>  Error: This repo is not allowlisted for Atlantis.

No idea. The people of Atlantis seem not to like this. 😄

**Comment by yshalabi** [comment]
> > Error: This repo is not allowlisted for Atlantis.
> 
> What is this!!?

quick... close it before they notice.

**Comment by witchel** [comment]
> > > Error: This repo is not allowlisted for Atlantis.
> > 
> > 
> > What is this!!?
> 
> quick... close it before they notice.

I welcome our new, aquatic overlords.

**Review by witchel — APPROVED** [review]
You have heard the pleas of children and your heart has softened.

File: libgraph/include/katana/PropertyGraph.h [hunk 1]
```diff
@@ -331,6 +331,14 @@ class KATANA_EXPORT PropertyGraph {
     return rdg_->part_metadata().policy_id_;
   }
 
+  /// \returns the current version of the graph (does not access storage)
+  Result<uint64_t> CurrentVersion() {
+    if (file_ == nullptr) {
```
Comment (amberhassaan) [inline libgraph/include/katana/PropertyGraph.h:336]: Minor: prefer `!file` instead.

**Pull request merged**